app/views/configuration/_ui_1.html.haml

```diff
@@ -46,8 +30,6 @@
                            @edit[:new][:display][:quad_truncate]),
                            :class    => "selectpicker")
 
-            :javascript
-              miqSelectPickerEvent("quad_truncate", "#{url}")
```

Comment:
https://bugzilla.redhat.com/show_bug.cgi?id=1650461

Fixed in https://github.com/ManageIQ/manageiq-ui-classic/pull/4942